floating-buttons.html

```diff
@@ -95,8 +95,8 @@
 			},
 
 			ready: function() {
-				this._container = Polymer.dom(this.root).querySelector('.container');
-				this._spacer = Polymer.dom(this.root).querySelector('.spacer');
+				this._container = Polymer.dom(this.root).querySelector('.vui-floating-buttons-container');
+				this._spacer = Polymer.dom(this.root).querySelector('.vui-floating-buttons-spacer');
```

Comment:
Cool.  Thanks.  I knew about the `$`.  Must have skimmed past `$$`... :)   Will switch.